sdk/framework/backend.go

```diff
@@ -528,9 +528,20 @@ func (b *Backend) handleRootHelp(req *logical.Request) (*logical.Response, error
 	// names in the OAS document.
 	requestResponsePrefix := req.GetString("requestResponsePrefix")
 
+	// Generic paths will primarily be used for code generation purposes.
+	// This will result in a dynamic mount path being placed instead of
+	// a hardcoded default path. For example /auth/approle/login would be replaced
+	// with /auth/{mountPath}/login. This will be replaced for all secrets
+	// engines and auth methods that are enabled.
+	genericMountPaths, ok := req.Get("genericMountPaths").(bool)
```

Comment:
Would this typecasting work if the API endpoint is called through curl (`sys/internal/specs/openapi`)?generic_mount_paths=true`. I'm assuming "true" here is a string and it would be typecast into a bool, which would result in `false`.

Reply:
Talked offline and this should not be a point since the underlying type of `req.Get("genericMountPaths")` is `bool`